gem/lib/phlexing/erb_transformer.rb

```diff
@@ -3,33 +3,46 @@
 require "deface"
 
 module Phlexing
-  class ErbTransformer
-    def self.transform(source)
-      transformed = source.to_s
-      transformed = transform_template_tags(transformed)
-      transformed = transform_erb_tags(transformed)
-      transformed = transform_remove_newlines(transformed)
-      transformed = transform_whitespace(transformed)
-
-      transformed
+  # Takes ERB and transforms it to Nokogiri-compatible HTML.
+  class ERBTransformer
+    def self.call(...)
+      new(...).call
     rescue StandardError
       source
     end
```

Comment:
Should this be removed?

Reply:
We should move the rescue to the `transform_erb_tags` method. I noticed in some cases (iirc if you provide it invalid ERB/Ruby) it will throw up.

Maybe we should add some tests with invalid/incomplete syntax

Reply:
In the case it throws, though, shouldn't we return an error to the user rather than trying to skip this step?

Reply:
It would definitely be better to show something to the user, yeah.

Reply:
Alright, I’ll remove the `rescue` and we can figure out the common errors and how to surface them.

Reply:
Awesome, sounds good!